editor/index.js

```diff
@@ -39,6 +39,9 @@ if ( dateSettings.timezone.string ) {
 window.jQuery( document ).on( 'heartbeat-tick', ( event, response ) => {
```

Comment:
Not specific to this pull request but the latter: jQuery is not defined as a dependency of `wp-editor` and we can only incidentally assume that it's present on the page. We should make this dependency explicit if we're going to call on it.

Reply:
The jQuery dependency is now explicit. Since we want to use heartbeat, we added hearbeat as a dependency in #2790 (https://github.com/WordPress/gutenberg/pull/2790/files#diff-620130744b6b73b822a3df609671d930R615). Heartbeat itself adds the jQuery dependency to the chain: https://github.com/WordPress/wordpress-develop/blob/5e9991f7e408244db336a858d948015a6801b6e3/src/wp-includes/script-loader.php#L151